### DISCUSSION TITLE
[IMPORTANT] 更新或者部署网页不生效请如图勾选清理缓存

### DISCUSSION THREAD

**Discussion by dreamhunter2333** [dreamhunter2333/cloudflare_temp_email#487] [Announcements]
<img width="997" alt="image" src="https://github.com/user-attachments/assets/d0a059fe-c3a5-485a-a4b0-39a9a136f69f">

**Comment by Albertjasson22** [comment]
I need a way to contact the developer or website owner as soon as possible.
